recipe/meta.yaml

```diff
@@ -1,44 +1,41 @@
-{% set version = "14.0.6" %}
-{% set prod_ver = '.'.join(version.split('.')[:3]) %}
+{% set version = "17.0.6" %}
+{% set major_ver = version.split('.')[0] %}
 
 package:
   name: compiler-rt-packages
   version: {{ version }}
 
 source:
   url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version.replace(".rc", "-rc") }}/llvm-project-{{ version.replace(".rc", "rc") }}.src.tar.xz
-  sha256: 8b3cfd7bc695bd6cea0f37f53f0981f34f87496e79e2529874fd03a2f9dd3a8a
+  sha256: 58a8818c60e6627064f312dbf46c02d9949956558340938b71cf731ad8bc0813
   patches:
     - patches/0001-no-code-sign.patch
     - patches/0002-compiler-rt-Make-7.0.0-compatible-with-10.9-SDK.patch
-    - patches/0003-set-cmake-paths.patch
-    - patches/0004-guard-use-of-FP_XSTATE_MAGIC1-based-on-glibc-version.patch
 
 build:
-  number: 0
+  number: 1
```

Comment:
Why do we need to bump the build num? It's a major version of the package.

Reply:
It keeps the feedstock more aligned with conda-forge's, which is useful when we're using this "sync with upstream" style of recipe upkeep.

Also, although this is more of an edge case: If a user has a certain channel priority and has configured both conda-forge and defaults as their channel, a higher build number will take precedence regardless of the channel. This prevents conda-forge's feedstock from unintentionally taking precedence in that case.